release/api/v1alpha1/bundle_types.go

```diff
@@ -235,6 +235,7 @@ type FluxBundle struct {
 type PackageBundle struct {
 	Version    string `json:"version,omitempty"`
 	Controller Image  `json:"packageController"`
+	Refreshser Image  `json:"tokenRefresher"`
```

Comment:
typo
```suggestion
	Refresher Image  `json:"tokenRefresher"`
```

Reply:
I would actually prefer to use `TokenRefresher` as the field name to make intentions clear.